cdc-sandbox/docker-compose.yml

```diff
@@ -41,15 +41,20 @@ services:
       - "8081:8081" # traefik dashboard
     networks:
       - nbs
+    environment:
+      - MODERNIZATION_UI_SERVER=${MODERNIZATION_UI_SERVER:-modernization-ui}
+      - MODERNIZATION_UI_PORT=${MODERNIZATION_UI_PORT:-80}
+      - MODERNIZATION_API_SERVER=${MODERNIZATION_API_SERVER:-modernization-api}
+      - MODERNIZATION_API_PORT=${MODERNIZATION_API_PORT:-8080}
     # volumes:
-    #   - ./proxy/:/etc/traefik/    
+    #   - ./proxy/:/etc/traefik/
```

Comment:
Is there a reason we're leaving commented out code in the file?  if so, can you document it in the file?  This is the same for all of the commented out code.

Reply:
It was pre-existing but that doesn't mean I can remove it now